lib/report/results-store.js

```diff
@@ -1,4 +1,11 @@
-var _ = require('lodash')
+var _assign = require('lodash/assign')
+var _each = require('lodash/each')
+var _get = require('lodash/get')
+var _last = require('lodash/last')
+var _map = require('lodash/map')
+var _omit = require('lodash/omit')
+var _set = require('lodash/set')
+var _some = require('lodash/some')
```

Comment:
Yeah. `assign`, `each`, `map`, and `some` are all native ES5 methods which would cut this to `get`/`set`, `last` and `omit`, which is much more palatable.